synfig-studio/src/gui/resources/CMakeLists.txt

```diff
@@ -1 +1,17 @@
 add_subdirectory(css)
+set(INSTALL_DESTINATION "share/synfig/resources")
```

Comment:
See that `add_subdirectory(css)`? Make a new **ui** folder and move all the glade files there.

Reply:
and call
```cmake
add_subdirectory(css)
add_subdirectory(ui)
```

Reply:
@Keyikedalube Understood !! I am on it :)

Reply:
Resloved :)